website/docs/reference/resource-configs/invalidate_hard_deletes.md

```diff
@@ -1,9 +1,17 @@
 ---
+title: invalidate_hard_deletes (legacy)
 resource_types: [snapshots]
 description: "Invalidate_hard_deletes - Read this in-depth guide to learn about configurations in dbt."
 datatype: column_name
+sidebar_label: invalidate_hard_deletes (legacy)
 ---
 
+:::warning This is a legacy config &mdash; Use the [`hard_deletes`](/reference/resource-configs/hard-deletes) config instead.
+
+In Versionless and dbt Core 1.9 and higher, the [`hard_deletes`](/reference/resource-configs/hard-deletes) config replaces the `invalidate_hard_deletes` config for better control over how to handle deleted rows from the source. 
+
+For new snapshots, set the config to `hard_deletes='invalidate'` instead of `invalidate_hard_deletes=true`. For existing snapshots, arrange an update of pre-existing tables before enabling this setting. Refer to 
```

Comment:
I'm guessing the final sentence that starts with "Refer to" is intended to link to something like the "Snapshot configuration migration" from https://github.com/dbt-labs/docs.getdbt.com/pull/6521?